packages/block-library/src/categories/edit.js

```diff
@@ -30,22 +29,14 @@ export default function CategoriesEdit( {
 	setAttributes,
 } ) {
 	const selectId = useInstanceId( CategoriesEdit, 'blocks-category-select' );
-	const { categories, isRequesting } = useSelect(
-		( select ) => {
-			const { getEntityRecords, isResolving } = select( coreStore );
-			const query = { per_page: -1, hide_empty: true, context: 'view' };
-			if ( showOnlyTopLevel ) {
-				query.parent = 0;
-			}
-			return {
-				categories: getEntityRecords( 'taxonomy', 'category', query ),
-				isRequesting: isResolving( 'getEntityRecords', [
-					'taxonomy',
-					'category',
-					query,
-				] ),
-			};
-		},
+	const query = { per_page: -1, hide_empty: true, context: 'view' };
+	if ( showOnlyTopLevel ) {
+		query.parent = 0;
+	}
+	const { categories, isResolving } = useEntityRecords(
```

Comment:
Should it be `records`?:
```suggestion
	const { records: categories, isResolving } = useEntityRecords(
```